timescale-forge/backup-restore-cloud.md

```diff
@@ -0,0 +1,20 @@
+# Backup and restore
+Timescale Cloud has a range of automated backup and restore mechanisms. All
+automated backups in Timescale Cloud are created using the `pgbackrest` tool.
+There is no need for you to manually perform backups for your Timescale Cloud
+service.
+
+Timescale Cloud automatically creates one full backup every week. We also create
```

Comment:
> The two most recent full backups are stored securely on our Timescale
 servers

We don't store them on our servers. We store backups on Amazon S3 service , which is supported by pgBackRest and provides redundancy and virtually unlimited storage.